BraveRewardsUI/README.md

```diff
@@ -1,10 +0,0 @@
-# brave-rewards-ios
```

Comment:
Don't need this file anymore?

Reply:
No, it has been replaced by `BraveRewards.resolved` in the BraveRewards folder